.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tick/third_party/cereal"]
```

Comment:
not sure but shouldn't this be lib/third_party/cereal ?

Reply:
probably, not sure it matters but I'll change it for consistency